dependencies.repos

```diff
@@ -0,0 +1,12 @@
+- git:
+    local-name: ros_industrial_cmake_boilerplate
+    uri: https://github.com/ros-industrial/ros_industrial_cmake_boilerplate.git
+    version: 0.7.4
+- git:
+    local-name: boost_plugin_loader
+    uri: https://github.com/tesseract-robotics/boost_plugin_loader.git
+    version: 0.3.2
+- git:
+    local-name: noether
+    uri: https://github.com/ros-industrial/noether.git
+    version: 548e5c2b70f9ee0a73da3d58563621b20c0d5710
```

Comment:
This version should be at least 1cd34d5c1b5cf488df03758c2808a2bbc86359e5 (two commits more recent) in order to include the parallel RICB version update.